src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -11,7 +11,7 @@
   <Import Project="..\..\build\Versioning.props" />
 
```

Comment:
@rossgrambo

/threadstart

Upgrading to netstandard2.1 is a big change. That means if anyone depends on us is netstandard2.0 the whole change needs to upgrade.

Reply:
Ah okay that makes sense. Should we instead be targeting both? As we should also be supporting netstandard2.1 for those customers?

Reply:
netstandard2.1 can reference netstandard2.1 packages.